docs/modules/ROOT/pages/how-tos/testing/integration/rerunning.adoc

```diff
@@ -1 +1,57 @@
-= Rerunning an integration test
\ No newline at end of file
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before you trigger subsequent tests.
+
+- The snapshot for which tests will be retriggered has completed its initial tests.
+
+- You have CLI access to the specific OpenShift cluster. For information on obtaining CLI access, refer Getting started.
+
+- You have an up-to-date kubectl binary.  Alternatively, the `oc` binary is also compatible.
+
+.Procedure
+
+. Identify the IntegrationTestScenario that needs to be rerun.  For a given snapshot, you can only rerun one IntegrationTestScenario.
+
+. Label the snapshot with `test.appstudio.openshift.io/run`, assigning the scenario name you wish to rerun as its value.
+
++
+[source]
+----
+$ kubectl label snapshot [snapshot name] test.appstudio.openshift.io/run=[scenario name]
+----
+
+. The tests are re-triggered automatically.  Once they are re-triggered, the system removes the label, allowing you to apply a new label for a different scenario if you wish to test multiple scenarios.
+
++
+[source]
+----
+$ tkn pipelinerun list
```

Comment:
There is no context given to this command. Is this just to verify that the pipeline was kicked off? Should a caveat be given that no triggered pipelinerun may be found if too much time has elapsed? Or that there may be more pipelineruns returned if more work is happening in the cluster?

Reply:
@dirgim Please let us know your thoughts?

Reply:
Agreed, both of those caveats make sense to me, something like:
```
Note: this command is only to verify that a re-triggered integration pipelineRun is started. If it's run a long time after the re-trigger command, it may not show any output since pipelineRun resources are pruned after completion. Additionally, the command may show other, unrelated pipelineRuns that are in flight at that same time.
```

Reply:
@MartinBasti fyi ^^